web/src/components/Minter.jsx

```diff
@@ -63,19 +63,22 @@ export default function Minter() {
   return (
     <div className="grid grid-cols-1 lg:grid-cols-2">
       {!isLoading && (
-        <ListItemImage typeId={0} rarityId={0} size="lg" grayscale={true} />
+        <ListItemImage 
```

Comment:
TODO: We should use a separate image component here since the gray placeholder is not in IPFS.

Reply:
Created an issue for this.